sktree/ensemble/_honest_forest.py

```diff
@@ -672,10 +672,7 @@ def _predict_proba(self, X, indices=None, impute_missing=None):
         zero_mask = posteriors.sum(2) == 0
         posteriors[~zero_mask] /= posteriors[~zero_mask].sum(1, keepdims=True)
 
-        if impute_missing is None:
-            pass
-        else:
-            posteriors[zero_mask] = impute_missing
+        posteriors[zero_mask] = impute_missing
```

Comment:
This implies that some of `posteriors` array is `None`, which would be bad as I don't think this is equivalent to `np.nan`. I'm surprised we don't run into any unit-test failures, but I guess this is a private function.

Instead, I would say: `if impute_missing is not None: posteriors[zero_mask] = impute_missing`.